Classifiers/classifier-Prisma_Cloud.json

```diff
@@ -1505,6 +1507,494 @@
                                 }
                         }
                 },
+		"GCP Compute Engine Misconfiguration": {
+                        "dontMapEventToLabels": false,
+                        "internalMapping": {
+                                "Account Name": {
+                                        "complex": {
+                                                "accessor": "account",
```

Comment:
Please fix in all files.

Reply:
@mayagoldb help!  I'm not sure what you mean? `Account Name` I think is correct.